web.main.js

```diff
@@ -28,24 +32,121 @@ module.exports = function (gulpWrapper, ctx) {
     function isFile(path) {
         var result = true;
         try {
-            fs.accessSync(path, fs.F_OK);                       
+            fs.accessSync(path, fs.F_OK);
         } catch (e) {
             result = false;
         }
         return result;
     }
 
+    /**
+     * Create bundles as was configured in gulp file
+     */
+    gulp.task('bundle-app', function (cb) {
+
+        if (ctx.bundleBuilderOn && ctx.bundleBuilderOn == true &&
```

Comment:
Although there is no formal convention, we usually name bool variables like "isBundleBuilderOn", "useBundleBuilder", "shouldBundle", "hasBundleBuilderOn", ...

Reply:
Already fixed to isBundleBuilderOn